backend/apps/readings/migrations/0005_auto_20190923_1529.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.2.4 on 2019-09-23 19:29
```

Comment:
You haven't modified the backend Django models in this PR at all, so something screwy is going on re: creating new migrations here. Something may have gone wrong when you merged from master at some point. These are going to conflict with the existing migrations -- please delete them from your branch, and just download the files from GitHub to make your backend/apps/readings/migrations folder identical to what's on the master branch.